src/amr/physical_models/mhd_model.hpp

```diff
@@ -44,12 +45,7 @@ class MHDModel : public IPhysicalModel<AMR_Types>
         resourcesManager->allocate(state, patch, allocateTime);
     }
 
-
-
-    virtual void
-    fillMessengerInfo(std::unique_ptr<amr::IMessengerInfo> const& /*info*/) const override
-    {
-    }
+    virtual void fillMessengerInfo(std::unique_ptr<amr::IMessengerInfo> const& info) const override;
```

Comment:
if there are to be no further subclasses, you don't need `virtual` - just `override`